src/ninemensmorris/NMMLogic.java

```diff
@@ -74,7 +74,7 @@
                         new InputStreamReader(
                                 getClass().getResourceAsStream("/ninemensmorris/resources/slot_index_ref.json")));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.

Reply:
This issue was fixed in a newer version, Thank ya kindly, By Giggity

Reply:
Thanks for your reply. Happy to help :)